components/payments/content/bat_payment_app.cc

```diff
@@ -0,0 +1,97 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/payments/content/bat_payment_app.h"
+
+#include <set>
+#include <string>
+
+#include "base/strings/stringprintf.h"
+#include "base/strings/utf_string_conversions.h"
+#include "brave/components/brave_rewards/common/constants.h"
```

Comment:
I don't think this file actually needs have an android guard in gn, but guards in gn ned matching guards in c++

Reply:
resolved: https://github.com/brave/brave-core/pull/7557/files#diff-119bf0ec63bc01e6eaf5c2cc1cfe3def4f6fad84912cb551300a5c68e827ac01